### ISSUE TITLE
Caribbean Admiral - Can't click on any destination or to town

### ISSUE THREAD

**Issue by desertking** [ruffle-rs/ruffle#12759]
### Describe the bug

Loading SWF file https://www.spiele-umsonst.de/azad/downloads/flashgames/carribeanadmiral.swf - Dunno if the error is related to the bug but I can't click on the map to get to any loation

ERROR core/src/display_object/movie_clip.rs:2183 Got "TypeError: Error #1010: A term is undefined and has no properties. (accessing field: BMenu)" when constructing AVM2 side of movie clip of type gui::Map

### Expected behavior

Click and get there

### Affected platform

Online demo

### Operating system

All

### Browser

All

**Comment by Lord-McSweeney** [comment]
Note that these two warnings are logged:
```
Serialization is not implemented for class other than Object: ScriptObject(ScriptObject { name: Objects::Story, ptr: 0xa0f190 })

Serialization is not implemented for class other than Object: ScriptObject(ScriptObject { name: Objects::Tutorial_Data, ptr: 0x11d8478 })
```
but I don't think they're relevant. No relevant stubs or errors are reported.

**Comment by Lord-McSweeney** [comment]
Those unimplemented serialization warnings likely cause this error spam when starting the game again:
```
Error dispatching event EventObject(EventObject { type: "enterFrame", class: flash.events::Event, ptr: 0xe1f230 }) to handler FunctionObject(FunctionObject { ptr: 0x470c810 }) : TypeError: Error #1010: A term is undefined and has no properties. (accessing field: b_loose)
	at Objects::AchievsManager/Update()
	at main/Update()
```
